Resolve dependencies clashes with Twig using twig-scoper

kagg-design opened the pull request:
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6746

Igor-WPML committed:
Resolve dependencies clashes with Twig using twig-scoper

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6746

zaantar committed:
Make the configuration file more generic and adjust scripts for Toolset and WPML accordingly. Update documentation.

zaantar commented:
@kagg-design / @Igor-WPML, I've pushed some changes that allow us to reuse a single config file. When I run it for Toolset, it doesn't produce any changes in the Twig library, so it should be safe. 

What do you think? Can you please try and let me know if it works for you?

Igor-WPML committed:
Fix problem on Windows Subsystem Linux (WSL). Setting of environment variable works, but it cannot be read in php. So, prefix is sent to php via --prefix option.

Fix directory separator problem on Windows.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6736

Igor-WPML committed:
Handle preg_replace_callback in CoreExtension.php

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6736

zaantar committed:
Update run-toolset.sh to synchronize it with the WPML script and update the readme.

Igor-WPML committed:
Fix array_walk_recursive() callback.

Small fixes in .sh

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6746

Igor-WPML committed:
Add mkdir for library to run-wpml-core.sh

Add run-wcml.sh

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6746
